grails-app/utils/org/pih/warehouse/LocalizationUtil.groovy

```diff
@@ -26,9 +26,41 @@ class LocalizationUtil {
         return localizationService.currentLocale
     }
 
+    /**
```

Comment:
An alternative to this would be to extend the Locale class and give it another constructor but I don't know if there's much advantage of that over what you've done